src/test/java/org/web3j/spring/autoconfigure/Web3jAutoConfigurationTest.java

```diff
@@ -98,8 +97,8 @@ public void testNoAdminClient() {
     public void testHealthCheckIndicatorDown() {
         load(EmptyConfiguration.class, "web3j.client-address=");
 
-        HealthIndicator web3jHealthIndicator = this.context.getBean(HealthIndicator.class);
-        Health health = web3jHealthIndicator.health();
+        Web3jHealthEndpointWebExtension healthEndpoint = this.context.getBean(Web3jHealthEndpointWebExtension.class);
+        Health health = healthEndpoint.health();
         assertThat(health.getStatus(), equalTo(Status.DOWN));
         assertThat(health.getDetails().get("error").toString(),
```

Comment:
This test fails, the error being returned is "org.web3j.exceptions.MessageDecodingException: Value must be in format 0x[1-9]+[0-9]* or 0x0".

Reply:
Tests seem to work for me, could it be a conflict in your env?

Reply:
Tests work fine, I had an instance of Test-RPC running on port 8545.